### PR TITLE
Check HolderTests under the proxy

### PR DESCRIPTION
**Pull request by ilyachur**
### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/plugins/proxy/src/compiled_model.hpp [hunk 1]
```diff
@@ -38,7 +38,10 @@ class CompiledModel : public ov::ICompiledModel {
     }
 
     ov::Any get_property(const std::string& name) const override {
-        return m_compiled_model->get_property(name);
+        auto property = m_compiled_model->get_property(name);
+        if (!property._so)
+            property._so = m_compiled_model._so;
+        return property;
```
Comment (songbell) [inline src/plugins/proxy/src/compiled_model.hpp:44]: in which case property need the so as well?

Comment (ilyachur) [reply src/plugins/proxy/src/compiled_model.hpp:44]: Yes because proxy returns the properties of the hw compiled model.

Comment (songbell) [reply src/plugins/proxy/src/compiled_model.hpp:44]: yes I understand, I mean why do the property need the so, generally they are of common types and hardware independent. maybe I'm missing something?

Comment (ilyachur) [reply src/plugins/proxy/src/compiled_model.hpp:44]: Any contains the shared pointer this can be allocated on the plugin side.

**Review by p-durandin — APPROVED** [review]

**Review by ilya-lavrenov — APPROVED** [review]

**Pull request merged**